### ISSUE TITLE
Minimize 'thin' pages

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#12]
For effective SEO, a website should contain a substantial amount of text content. So-called 'thin' pages with very little content are unlikely to rank, or contribute value to a website.

This check highlights pages with fewer than 300 words, and no significant non-text content (e.g. videos or plugins). 300 words is the consensus estimate for what Google considers 'thin'.

This is not a hard rule. You will sometimes have valid reasons for pages to have very little text content, for example:

To show a large image or video
For a progress screen ("Please wait")
For maps or other content which lacks equivalent machine-readable text
In cases like these, you can choose to ignore the page manually.
